.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
 max-line-length = 100
+enable-extensions = L101,L102
```

Comment:
It's enabled by default so we don't need this line, but it doesn't hurt either.

Reply:
Let me test some more.  I added `off_by_default = True` in the class.

Reply:
:+1:

Reply:
I tested this. Need to explicitly add `enable-extensions = L101,L102` or `enable-extensions = L` in .flake8 config to enable this plugin.

Reply:
Yeah, I originally missed ``off_by_default`` line so all is well.